addendum.dict

```diff
@@ -339,13 +351,18 @@ Schumer	𐑖𐑵𐑥𐑼
 Schwarzenegger	𐑖𐑢𐑹𐑑𐑕𐑩𐑯𐑧𐑜𐑼
 Secheron	𐑕𐑧𐑖𐑼𐑪𐑯
 Servox	𐑕𐑺𐑝𐑪𐑟
+Sega 𐑕𐑧𐑜𐑩
```

Comment:
The pronunciation of this name varies widely across accents. To conform with contemporary 'rhotic Received Pronunciation', I'd suggest 𐑕𐑱𐑜𐑩 (based on the Cambridge Pronouncing Dictionary).